modules/core/shared/src/main/scala/scalacache/package.scala

```diff
@@ -3,6 +3,7 @@ import scala.language.higherKinds
 
 package object scalacache {
 
+  //todo
```

Comment:
Is this todo indicating that `type Id[X] = X` is able to be removed? I am guessing that it can be removed in favor of people using cats Id when appropriate. Although it won't be usable for any of the algebras requiring Sync/Async anyway..

Reply:
Yes I think this can be removed. I'll add a commit to do that.